hodor

```diff
@@ -44,21 +50,47 @@ class Hodor
     conf = YAML.load(ERB.new(File.read('.hodorfile')).result)
     @containers = conf['containers']
     @tasks = conf['tasks']
+    @hostmanager = "boot2docker"
+    @host = "boot2docker-vm"
+    @ssh_file = "~/.ssh/id_boot2docker"
```

Comment:
Now, let move those defaults (hostmanager, host and ssh_file) out of `process_project_config` method, since they do not relay on config and move them to regular vars in `initialize` method on top. Then we will see which vars we have at glance. Almost there, Joachim!